docs/using.rst

```diff
@@ -114,22 +69,23 @@ To use ADB with Host-Based tools:
     config.servicemanager.services = 'openshift'
 
    **Note:**
-* Docker is a default service for the ADB and does not require any configuration to ensure it is started.
-  Additionally, the Red Hat Enterprise Linux Container Development Kit, which is
-  based on the Atomic Developer Bundle, automatically starts OpenShift as well.
-* You can enable multiple services as a comma separated list. For instance: `docker, openshift`.
 
-3. Enable any specific options for the services you have selected as:
+   * Docker is a default service for the ADB and does not require any configuration to ensure it is started.
+     The Red Hat Enterprise Linux Container Development Kit, which is based on
+     the Atomic Developer Bundle, automatically starts OpenShift.
+   * You can use a comma separated list to enable multiple services. For instance: `docker, openshift`.
+
+3. Enable any specific option for the services you have selected in the Vagrantfile as:
 
    For instance, in OpenShift, specific versions can be specified using the following variables:
 
    1. ``config.servicemanager.openshift_docker_registry = "docker.io"`` - Specifies the registry from where the service should be pulled.
    2. ``config.servicemanager.openshift_image_name = "openshift/origin"`` - Specifies the image to be used.
    3. ``config.servicemanager.openshift_image_tag = "v1.2.0"`` - Specifies the version of the image to be used.
 
-4. Start the ADB using ``vagrant up``. For details consult the `Installation documentation`_.
+4. Start the ADB using ``vagrant up``. For details, consult the `Installation document`_.
```

Comment:
is it necessary to send users back to the installation document just to run one command? this sentence makes it seem like there's more to this step than just running "vagrant up"